tests/quarantine/quarantine_unit_test.py

```diff
@@ -0,0 +1,66 @@
+
+import pytest
+
+from twister2.filter.quarantine_filter import QuarantineData
+from twister2.exceptions import TwisterConfigurationException
+
+
+def test_quarantine_load_data(tmp_path):
+    quarantine_file = tmp_path / "quarantine.yml"
+    quarantine_file.write_text("""
```

Comment:
You can use `textwrap.dedent` to have indents in the text (there are some usage in the code already).